### PR TITLE
Export functions in sd-device and sd-hwdb

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/libsystemd/libsystemd.sym [hunk 1]
```diff
@@ -576,4 +576,64 @@ global:
         sd_bus_message_readv;
         sd_bus_set_method_call_timeout;
         sd_bus_get_method_call_timeout;
+
+        sd_device_ref;
+        sd_device_unref;
+
+        sd_device_new_from_syspath;
```
Comment (yuwata) [inline src/libsystemd/libsystemd.sym:583]: We have never add so many blank lines...

**Comment by yuwata** [comment]
Fedora CI fails by test-sd-hwdb.

File: src/libsystemd/libsystemd.sym (hunk 1 above)
Comment (keszybz) [reply src/libsystemd/libsystemd.sym:583]: Yeah. But I think it makes the whole thing a bit more readable... I kept the same structure as in the .h file.

**Comment by yuwata** [comment]
@keszybz I noticed that `*_unrefp()` are inline functions. They are dropped by #10239.

**Pull request merged**